src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Source.cs

```diff
@@ -86,6 +86,11 @@ private static async Task<Checksum> ComputeSourceSymbolsChecksumAsync(ProjectSta
                 allChecksums.Add(compilationOptionsChecksum);
                 allChecksums.Add(parseOptionsChecksum);
 
+                // Include serialization format version in our checksum.  That way if the 
+                // version ever changes, all persisted data won't match the current checksum
+                // we expect, and we'll recompute things.
+                allChecksums.Add(SerializationFormatChecksum);
```

Comment:
💭 It seems like it would be appropriate to push this all the way to the serialization layer. Not sure if it's possible to do that efficiently though.

Reply:
i'm interested in what you're asking for here. Can you clarify this a bit so we're on hte same page?